addons/point_of_sale/static/src/app/components/popups/product_info_popup/product_info_popup.xml

```diff
@@ -49,54 +48,49 @@
             <div class="financials-order d-grid gap-2 mt-3" style="grid-template-columns: repeat(2, 1fr);">
                 <div class="section-financials text-start">
                     <h3 class="section-title">Financials</h3>
-                    <div class="section-financials-body d-flex flex-column">
-                        <table class="table table-borderless mb-0">
-                            <tr>
-                                <td>Price excl. Tax:</td>
-                                <td><t t-esc="env.utils.formatCurrency(props.info.productInfo.all_prices.price_without_tax)"/></td>
-                            </tr>
-                            <tr t-if="_hasMarginsCostsAccessRights()">
-                                <td>Cost:</td>
-                                <td><t t-esc="props.info.costCurrency"/></td>
-                            </tr>
-                            <tr t-if="_hasMarginsCostsAccessRights()">
-                                <td>Margin:</td>
-                                <td><t t-esc="props.info.marginCurrency"/> (<t t-esc="props.info.marginPercent"/>%) </td>
-                            </tr>
-                        </table>
-                        <table class="table table-borderless">
-                            <t t-foreach="props.info.productInfo.pricelists" t-as="pricelist" t-key="pricelist.name">
-                                <tr>
-                                    <td t-esc="pricelist.name"/>
-                                    <td t-esc="env.utils.formatCurrency(pricelist.price)"/>
-                                </tr>
-                            </t>
-                        </table>
+                    <div class="grid gap-2">
+                        <label class="fw-bold">Price excl. Tax:</label>
+                        <span><t t-esc="env.utils.formatCurrency(props.info.productInfo.all_prices.price_without_tax)"/></span>
+                        <label class="fw-bold">VAT:</label>
+                        <span>
+                            <t t-esc="props.info.taxAmount"/>
+                            (<t t-esc="props.info.taxName"/>)
+                        </span>
+                        <label class="fw-bold">Price incl. Tax:</label>
+                        <span><t t-esc="env.utils.formatCurrency(props.info.productInfo.all_prices.price_with_tax)"/></span>
+                        <label class="fw-bold">Cost:</label>
+                        <span><t t-esc="props.info.costCurrency"/></span>
+                        <label class="fw-bold">Margin:</label>
```

Comment:
Can you check `_hasMarginsCostsAccessRights` seems to be forgotten at some places